src/migrations/20210305172128-add-grant-goal-unique-index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  up: async (queryInterface) => {
+    queryInterface.addIndex('GrantGoals', ['grantId', 'goalId', 'granteeId'], { unique: true });
```

Comment:
Is there a benefit to including `granteeId` in this index? I would think that the `grantId, goalId` combo alone would guarantee the same thing.

Reply:
Yeah `granteeId` isn't needed, removing.